src/components/ReferenceData.vue

```diff
@@ -131,7 +133,6 @@ export default {
 		placeholder: { type: String, required: false, default: null },
 		actions: { type: Array, default: ()=>[] },
 		disableInternalSearch: { type: Boolean, default: false },
-		disabled: { type: Boolean, default: false },
```

Comment:
was this on purpose?

Reply:
yes, to make it consistent with other components after adding the readOnly prop to base.vue